module/Application/src/Controller/StatusController.php

```diff
@@ -0,0 +1,172 @@
+<?php
+
+namespace Application\Controller;
+
+use Application\Library\ApiProblem\ApiProblem;
+use Application\Library\ApiProblem\ApiProblemException;
+use Application\Library\Authorization\UnauthorizedException;
+use Application\Library\DateTime;
+use Application\Library\Http\Response\Json;
+use Application\Model\DataAccess\Repository\Application\LockedException;
+use Application\Model\Service\Applications\Service;
+use Exception;
+use Opg\Lpa\DataModel\Lpa\Lpa;
+use Application\Model\Service\ProcessingStatus\Service as ProcessingStatusService;
+use Zend\Mvc\Controller\AbstractRestfulController;
+use Zend\Mvc\MvcEvent;
+use ZF\ApiProblem\ApiProblemResponse;
+use ZfcRbac\Service\AuthorizationService;
+
+class StatusController extends AbstractRestfulController
+{
+    /**
+     * @var string
+     */
+    protected $identifierName = 'lpaIds';
```

Comment:
@SeemaMenon Is `$identifierName` used anywhere?

Reply:
@nsmithuk just rechecked and $identifierName is not needed actually, but when started writing the code the understanding was $identifierName helped with the binding. I shall take it out and `along with a new unit test shall create a new PR shortly